Cargo.toml

```diff
@@ -16,4 +16,38 @@ opt-level = 1
 opt-level = 3
 
 [dependencies]
-bevy = "0.12"
+bevy = "0.14"
+# Required for web builds.
+wasm-bindgen = { version = "0.2", optional = true }
+
+[features]
+default = [
+    # Default to a native dev build.
+    "native_dev",
+]
+core = []
```

Comment:
Add a comment for what sort of features go here

Reply:
Don't need this feature in retrospect, removed.